RELEASE_NOTES.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## v1.1.0
+* Netavark no longer starts Aardvark bound to port 53, but instead uses a high port and forwards traffic from 53 to this port using firewall rules. This allows the host to use Port 53 without conflict.
```

Comment:
I thought the default is still 53 and you have to change it in cotnainers.conf?

Reply:
Yes https://github.com/containers/netavark/blob/eccad5c2235d38d239ff24a7c9044ed245c56fc2/src/commands/setup.rs#L74-L85

There is no way to choose a random one without storing it somewhere.

Reply:
I mean this should not be part of netavark release notes, netavark uses NETAVARK_DNS_PORT or defaults to 53.
If podman overwrites that we should document it there.

Please change it

Reply:
Repushed. I want to mention this here since the core functionality is in Netavark and we should have a pointer that v1.1.0 is the earliest supporting version, but I changed the release note to make it clear this is optional.